src/BizHawk.Emulation.Cores/Arcades/MAME/MAME.cs

```diff
@@ -174,21 +92,21 @@ private void ExecuteMAMEThread()
 				, "-skip_gameinfo"                      // forbid this blocking screen that requires user input
 				, "-nothrottle"                         // forbid throttling to "real" speed of the device
 				, "-update_in_pause"                    // ^ including frame-advancing
-				, "-rompath",            _gameDirectory // mame doesn't load roms from full paths, only from dirs to scan
+				, "-rompath",                      ""   // mame doesn't load roms from full paths, only from dirs to scan
 				, "-joystick_contradictory"             // allow L+R/U+D on digital joystick
 				, "-nonvram_save"                       // prevent dumping non-volatile ram to disk
-				, "-artpath",          "mame\\artwork"  // path to load artowrk from
-				, "-diff_directory",      "mame\\diff"  // hdd diffs, whenever stuff is written back to an image
+				, "-artpath",           "mame/artwork"  // path to load artowrk from
+				, "-diff_directory",       "mame/diff"  // hdd diffs, whenever stuff is written back to an image
 				, "-cfg_directory",                "?"  // send invalid path to prevent cfg handling
 				, "-volume",                     "-32"  // lowest attenuation means mame osd remains silent
 				, "-output",                 "console"  // print everything to hawk console
 				, "-samplerate", _sampleRate.ToString() // match hawk samplerate
+				, "-sound",                     "none"  // forbid osd sound driver

```

Comment:
Are we sure samplerate is not butchered?
https://github.com/mamedev/mame/pull/10483 isn't pulled yet.

Reply:
I removed the problematic code in the last commit I did.